tests/rkt_app_isolator_test.go

```diff
@@ -28,6 +28,7 @@ import (
 const (
 	// if you change this you need to change tests/image/manifest accordingly
 	maxMemoryUsage = 25 * 1024 * 1024 // 25MB
+	CPUQuota       = 100              // milli-cores
```

Comment:
What about using 800 and changing the manifest accordingly? Maybe the tests will run faster?